package.json

```diff
@@ -23,13 +24,38 @@
     "promise",
     "cordova-sqlite-storage"
   ],
-  "author": "Andrzej Porebski",
+  "files": [
+    "lib/*",
+    "platforms/*"
+  ],
+  "author": "Mendix",
```

Comment:
I think we need to keep the original author as well, no?

Reply:
Added